src/coreclr/vm/interpexec.cpp

```diff
@@ -1,10 +1,11 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-#include <math.h>
-
 #ifdef FEATURE_INTERPRETER
 
+#include "threads.h"
+#include "gcenv.h"
+#include <math.h>
```

Comment:
What do we need the math.h for?

Reply:
I think I was using a math function previously, I'll try removing it